build-in-container/scripts/build-mariner.sh

```diff
@@ -87,8 +87,11 @@ done
 source /mariner/scripts/setup.sh
 
 if [[ "${container_type}" == "build" ]]; then
-    # exit if SPECS/ is empty
-    if [ ! "$(ls -A $SPECS_DIR)" ]; then exit; fi
+    # exit if SPECS/ is empty && container_type is build as there is nothing to be done
+    if [ ! "$(ls -A $SPECS_DIR)" ]; then
+        echo -e "-------- \033[31m ALERT: Exiting the build container, found no specs to build \033[0m --------"
```

Comment:
Please add the $SPECS_DIR as well to the error message, so the user knows which directory didn't have spec files populated. This could help the user to diagnose the problem faster by providing relevant context behind why the error happened.

Reply:
Thanks for making this update! Couple recommendations:
1. Can we condense and keep the message to 1 line instead of 2?
2. Instead of prescribing the fix to add specs to the folder, thoughts on just making this an informative error message (i.e., "no files found in $SPECS_DIR").

Reply:
The $SPECS_DIR inside container is different from $SPECS_DIR on host. That's why the echo statement tells user to add specs to $mariner_dir/SPECS (User knows what $mariner_dir means, but not what $SPECS_DIR means). I can add a more verbose echo that says "no files found in $SPEC_DIR which is mounted at $mariner_dir/SPECS on the host". 
This would be true, if the user has not changed mount points. The script doesn't give user options to change mountpoints, the only way to change it is to edit the script.

Reply:
I like your more verbose echo better. Users have a habit of finding ways to break things in ways we do not anticipate, so having the error message be informative but not prescriptive will help catch those